docs/keystone-auth.md

```diff
@@ -16,6 +16,10 @@ For example, an administrator may execute the command:
 
 To grant the Openstack project with that UUID access to HaaS.
 
+Note that the plugin recognizes any user with an `admin` role on any
+project as a HaaS administrator, similar to the default policy for core
```

Comment:
could you say `as a global HaaS administrator, able to administrate all projects,`